contrib/mkimage-yum.sh

```diff
@@ -30,7 +30,9 @@ if [ -f /etc/dnf/dnf.conf ] && command -v dnf &> /dev/null; then
 	yum_config=/etc/dnf/dnf.conf
 	alias yum=dnf
 fi
-install_groups="Core"
+install_groups=('Core')
+# for names with spaces, use double quotes (") as install_groups=('Core' '"Compute Node"')
+install_packages=('')
```

Comment:
This should default to being empty instead, ala:
```suggestion
install_packages=()
```

Reply:
Will make these changes.